apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -1,15 +1,16 @@
+import moment from 'moment';
 import {createSlice, PayloadAction, createAsyncThunk} from '@reduxjs/toolkit';
+import {LabState} from '@cdo/apps/lab2/lab2Redux';
+const registerReducers = require('@cdo/apps/redux').registerReducers;
+
+import {initialChatMessages} from '../constants';
+import {getChatCompletionMessage} from '../chatApi';
```

Comment:
I reorganize imports so the absolute ones are at the top and the relative ones are below, sorry for the extra diff 🙈